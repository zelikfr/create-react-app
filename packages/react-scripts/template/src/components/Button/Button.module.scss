@import 'styles/variables';

.button {
    display: inline-block;
    padding: 10px 5px;
    border-radius: 5px;
    border: 1px solid $primaryColor;
    color: $altColor;
}