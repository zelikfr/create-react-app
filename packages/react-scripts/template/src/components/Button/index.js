import React from 'react'
import styles from './styles.scss'

const Button = props => (
    <button {...props} className={styles.button} />
)

export default Button