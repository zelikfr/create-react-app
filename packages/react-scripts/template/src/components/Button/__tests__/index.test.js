import React from 'react'
import { shallow } from 'enzyme'
import Button from '../'

describe('Button', () => {
    it('should create a Button instance', () => {
        // When I render the Button element
        const component = shallow(<Button />)

        // Then I should have the component to match the snapshot.
        expect(component).toMatchSnapshot()
    })
})
