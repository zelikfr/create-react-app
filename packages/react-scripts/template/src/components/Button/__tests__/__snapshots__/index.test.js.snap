// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should create a Button instance 1`] = `
<button
  className="button"
/>
`;
